.github/PULL_REQUEST_TEMPLATE/new_source.md

```diff
@@ -0,0 +1,23 @@
+---
+name: New Python Source Connector
```

Comment:
Maybe the file should be called new_python_source

Reply:
agree, changed